setup.py

```diff
@@ -15,6 +15,7 @@
     'easy-thumbnails',
     'phonenumbers',
     'vobject',
+    'sphinx-rtd-theme',
```

Comment:
We should require this somewhere else, as not everybody installing the package wants to pull in the whole sphinx dependencies.